tests/evaluate/test_evaluation.py

```diff
@@ -569,23 +569,6 @@ def test_pandas_df_regressor_evaluation(linear_regressor_model_uri):
         assert v == saved_metrics[k]
 
 
-def test_pandas_df_regressor_evaluation_without_targets(linear_regressor_model_uri):
```

Comment:
don't need this test anymore since we have `test_targets_is_required_for_regressor_and_classifier_models`